Python/Product/PythonTools/PythonTools/OutliningTaggerProvider.cs

```diff
@@ -323,7 +322,7 @@ class OutliningWalker : PythonWalker {
             }
 
             public override bool Walk(ClassDefinition node) {
-                AddTagIfNecessary(node, node.HeaderIndex, node.Decorators);
+                AddTagIfNecessary(node, node.HeaderIndex + 1, node.Decorators);

```

Comment:
Is HeaderIndex inconsistent between ClassDef and FunctionDef? Or does FunctionDef already add 1?